src/app/route/CartPage/CartPage.style.scss

```diff
@@ -18,8 +18,8 @@
 
 .CartPage {
     @include before-desktop {
-        padding-bottom: calc(var(--navigation-tabs-height) + 130px);
-        margin-bottom: 0;
+        padding-bottom: 113px;
```

Comment:
If this value is calculated from some other value, please do that calculation here. Otherwise, everything seems legit.